lib/fortnox/api/models/invoice.rb

```diff
@@ -1,142 +1,43 @@
-require "fortnox/api/models/base"
+require "fortnox/api/models/document_base"
 require "fortnox/api/models/edi_information"
-require "fortnox/api/models/email_information"
-require "fortnox/api/models/row"
+require "fortnox/api/models/invoice_row"
```

Comment:
We have `Fortnox::API::Model::Row` as a superclass for `Rows`. Maybe we should rename it to `DocumentRow` now? Then we have both `InvoiceRow` and `OrderRow` that inherits from `Row`. Maybe we should get rid of this inheritance also and use a module again?